testing/0006-no-ascii-chars

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+source "$TESTS_DIR/functions.sh"
+
+msgfile="$TEST_DIR/message"
+
+cat >"$msgfile" <<EOF
+This is the message with non-ascii characters
+
+This is the description with non-ascii characters (èé).
+EOF
+
+# Copy out the cover letter before git-send-email(1) is invoked
+hookfile=".git/hooks/pre-publish-send-email"
+coverfile="$TEST_DIR/0000-cover-letter-patch"
+cat >"$hookfile" <<EOF
+#!/bin/bash
+cp "\$1/0000-cover-letter.patch" "$coverfile"
+exit 0
+EOF
+chmod 755 "$hookfile"
+
+git checkout -q -b mybranch
+
+GIT_EDITOR="cp $msgfile" git-publish --no-inspect-emails \
+  --to somebody@example.com \
+  -b HEAD^ \
+  --cover-letter \
+  --subject-prefix="PATCH ò" || :
+
+echo -ne \
+    'Subject: [PATCH ò 0/1] This is the message with non-ascii characters\n' \
+    >"$TEST_DIR/expected"
+grep '^Subject:' "$coverfile" >"$TEST_DIR/found"
+assert diff -u "$TEST_DIR/expected" "$TEST_DIR/found"
```

Comment:
...and a full diff of the message - or a diff that covers the from, subject, and body if there are unpredictable timestamps/version numbers in the output.

Reply:
Sure!